lib/embulk/input/marketo/timeslice.rb

```diff
@@ -39,6 +88,7 @@ def transaction(config, &control)
               encryption_key: config.param(:encryption_key, :string),
               from_datetime: from_datetime,
               to_datetime: to_datetime,
+              workers: config.param(:workers, :integer, default: 5), # prime number is better for shuffling fetch time range in each task
```

Comment:
I think that `workers` parameter is unnecessary. the # of workers is categorized on executor plugin. input plugin should not consider about it.